dev/io.openliberty.microprofile.config.3.1.internal_fat_tck/fat/src/io/openliberty/microprofile/config/internal_fat_tck/Config31TCKLauncher.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.config.internal_fat_tck;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.AllowedFFDC;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.tck.TCKResultsInfo.Type;
+import componenttest.topology.utils.tck.TCKRunner;
+
+/**
+ * This is a test class that runs a whole Maven TCK as one test FAT test.
+ * There is a detailed output on specific
+ */
+@RunWith(FATRunner.class)
+@Mode(TestMode.LITE)
+public class Config31TCKLauncher {
```

Comment:
Should the MinimumJavaLevel annotation be added here?  Or possibly add `fat.minimum.java.level: 11` to the bnd file.

Reply:
I am adding `@MinimumJavaLevel` but actually I think that since the bnd already has `javac.target: 11`, that means that `fat.minimum.java.level` gets defaulted to 11 as well.